fe/fe-core/src/main/java/com/starrocks/analysis/BoolLiteral.java

```diff
@@ -61,11 +61,6 @@ protected BoolLiteral(BoolLiteral other) {
         value = other.value;
     }
 
-    @Override
-    public Object getRealValue() {
```

Comment:
Because the definition here is to return the Object in Literal, and it can directly correspond to the object in ConstantOperator. getRealObjectValue can better reflect the meaning of "Object"